iterx.go

```diff
@@ -206,9 +206,9 @@ func (iter *Iterx) scanAll(dest interface{}) bool {
 
 // isScannable takes the reflect.Type and the actual dest value and returns
 // whether or not it's Scannable. t is scannable if:
-//   * ptr to t implements gocql.Unmarshaler, gocql.UDTUnmarshaler or UDT
-//   * it is not a struct
-//   * it has no exported fields
+//   - ptr to t implements gocql.Unmarshaler, gocql.UDTUnmarshaler or UDT
+//   - it is not a struct
+//   - it has no exported fields
```

Comment:
Why this change? Asterisks are valid Markdown itemizers as well AFAIK.

Reply:
This one is gone, it is already in master, reason why it have changed is `golangci-lint` does not tollerate anything by `-` there.